### PR TITLE
(chore) - add prettier + pre-commit hooks

### LINKED ISSUE #56: Setup Prettier!
To avoid unnecessary time time spent in PRs for things like this

image

### PR DESCRIPTION
**Pull request by JoviDeCroock**
Now every time before a commit occurs prettier will run over the files you're staging. If anyone has an opinion I'd be happy to hear!

CC @Dhaiwat10 sorry I just see you made an issue about this 😅 

Resolves #56

Y'all will probably need to run `yarn` on your own machine to enable them if you already have stuff installed 😅

### CODE REVIEW

**Comment by Dhaiwat10** [comment]
> CC @Dhaiwat10 sorry I just see you made an issue about this 😅

Oh no, thank you!

**Review by Dhaiwat10 — APPROVED** [review]
LGTM :))

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
```
Comment (frankTurtle) [inline .prettierrc.json:2]: 🙏

**Review by frankTurtle — APPROVED** [review]
I looked at 30% of the files, then saw your comment about how, this being the first, it's going to touch A LOT. 

Everything looks solid and *pretty* to me 💪

**Pull request merged**